### PR TITLE
feat(breadcrumb-harmonised): add component - INNO-1817

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

specs: https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Harmonised+Breadcrumb+-+Design+Specifications

playground: https://deploy-preview-1340--europa-component-library.netlify.com/playground/ec/?path=/story/components-navigation-breadcrumb-harmonised--group-1

note: the component isn't dipslayed on the website as is, as it will be part of the page header

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit c3bea6291554057f46cc63762c6f3df42e443f21

https://deploy-preview-1340--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit b8138869b3430930f23572be7f8efc9bf83b534a

https://deploy-preview-1340--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit a3d715d66f7ea1beaf7e62e5948e3a582280b1de

https://deploy-preview-1340--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 17d6d0e6ee2623fb2af6c0e61822393f22ae6a15

https://deploy-preview-1340--europa-component-library.netlify.com

File: .size-limit.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   {
     "path": "dist/packages/ec-preset-legacy-website/styles/ecl-ec-preset-legacy-website.css",
     "webpack": false,
-    "limit": "23 KB"
+    "limit": "24 KB"
```
Comment (yhuard) [inline .size-limit.json:37]: 😭

**Review by yhuard — APPROVED** [review]

**Pull request merged**